osbs/core.py

```diff
@@ -134,11 +140,12 @@ def _build_k8s_url(self, url, _prepend_namespace=True, **query):
             url += ("?" + urlencode(query))
         return urljoin(self.k8s_api_url, url)
 
-    def _build_url(self, url, _prepend_namespace=True, **query):
+    def _build_url(self, api_version, url, _prepend_namespace=True, **query):
```

Comment:
Should it infer this by looking up the first part of `url` in `OCP_RESOURCE_API_VERSION_MAP`?

Reply:
Oh, I see, /watch/... breaks that.